packages/language-support/src/parserWrapper.ts

```diff
@@ -68,52 +91,78 @@ export type LabelOrRelType = {
   };
 };
 
-export interface EnrichedParsingResult extends ParsingResult {
-  diagnostics: SyntaxDiagnostic[];
-  stopNode: ParserRuleContext;
-  collectedLabelOrRelTypes: LabelOrRelType[];
-  collectedVariables: string[];
-  collectedCommands: ParsedCommand[];
-}
-
-export interface ParsingScaffolding {
-  query: string;
-  parser: CypherParser;
-  tokenStream: CommonTokenStream;
-}
-
 export function createParsingScaffolding(query: string): ParsingScaffolding {
   const inputStream = CharStreams.fromString(query);
   const lexer = new CypherLexer(inputStream);
   const tokenStream = new CommonTokenStream(lexer);
-  const parser = new CypherParser(tokenStream);
-  parser.removeErrorListeners();
+  const stsTokenStreams = splitIntoStatements(tokenStream, lexer);
```

Comment:
`sts` short for statements? you don't need change it, it just tripped me up a little since I think we've shortened it `stmts` other places in the code base